docs/TUTORIAL.md

```diff
@@ -1,3 +1,305 @@
 # Tutorial
 
-Coming soon!
+The **ksonnet** [readme][readme] 
```

Comment:
But this assumes we know that the reader worked through the readme example first. We can't know. (Another reason for simple present)